hs_core/templates/pages/metadata_terms.html

```diff
@@ -108,14 +108,28 @@
             <li><strong>Description: </strong> a text statement that points to a rights
                 management or usage statement for the resource.</li>
         </ul>
-
+        <a name="isCopiedFrom"></a>
         <ul>
             <li><strong>Term: </strong>URL</li>
             <li><strong>URI: </strong><a href="#URL">http://www.hydroshare.org/terms/URL</a></li>
             <li><strong>Label: </strong>Uniform Resource Locator</li>
             <li><strong>Description: </strong>URL that points to a rights
                 management or usage statement for the resource.</li>
         </ul>
+        <a name="isHostedBy"></a>
```

Comment:
@pkdash at the beginning, I also got confused as what I expected to see is the name = "isCopiedFrom" then follow that everything related to this term. I did that to follow the same pattern of the defined metadata terms for generic resource and I found a reason for that. I did a test to see what is the different between putting the  name="isCopiedFrom" immediately before other lines that describe this term or shift it to the beginning of the previous term description. I found that it is better for visualization to put the  name="isCopiedFrom" at the beginning of the previous term description (as I did here) as if you call now the URL related to this term it will show all the information about it. See the difference below:
![image](https://cloud.githubusercontent.com/assets/5543781/19311191/3cec06fc-905b-11e6-824f-6369157cbe93.png)


![image](https://cloud.githubusercontent.com/assets/5543781/19311231/7150ed22-905b-11e6-83cc-2246bd4edf55.png)